packages/website/components/blog/cards.js

```diff
@@ -0,0 +1,110 @@
+import Button from '../button'
+import Link from 'next/link'
+import React from 'react'
+import Tags from '../tags'
+import countly from '../../lib/countly'
+
+// custom styles from /styles/blog.css
+
+/**
+ * Blog Card Component
+ *
+ * @param {Object} props
+ * @param {import('../types').PostMeta} props.post
+ * @param {() => void} props.onClick
+ * @returns {JSX.Element}
+ */
+
+export const Card = ({ post, onClick }) => (
+  <Link href={`/blog/post/${post.slug}`}>
+    {/* 👇 because next/link won't accept onClick */}
+    <a
```

Comment:
Maybe a generic container element instead so we don't end up with nested anchor tags?

Reply:
next/link does not duplicate child `a`